### PR TITLE
CI improvements after parachain merge

### PR DESCRIPTION
**Pull request by serhii-temchenko**
## Describe your changes
After merge of parachain - CI/CD should be extended with inclusion of parachain.

For the reference - parachain merge implemented in PR #870.

## Issue ticket number and link
[241](https://github.com/Polkadex-Substrate/issue-tracker/issues/241)

## Checklist before requesting a review
- [X] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [X] I removed all Clippy and Formatting Warnings. 
- [X] I added required Copyrights.

:warning: FIXED (partially in destination PR, partially in this PR) ~~Currently CI is failing since PR #870 is not ready to be merged because of multiple compilation errors which should be fixed before merge of this PR by @zktony.~~ FYI @pavankanteti.

~~Current issue is described in the comment below.~~ fixed as well.

### CODE REVIEW

**Comment by serhii-temchenko** [comment]
Currently everything ready to be merged. CI is fails because of credentials issue. Reported to devops team.
@pavankanteti FYI.

**Comment by serhii-temchenko** [comment]
> Currently everything ready to be merged. CI is fails because of credentials issue. Reported to devops team. @pavankanteti FYI.

Now it was fixed. Currently waiting on CI to pass and after that could be merged to the destination branch.

**Review by zktony — APPROVED** [review]

**Pull request merged**
